### PR TITLE
feat: add tfnotify plugin

### PR DESCRIPTION
**Pull request by jnavarrof**
## Summary

Description:

- Tool repo URL: https://github.com/mercari/tfnotify
- Plugin repo URL: https://github.com/jnavarrof/asdf-tfnotify

## Checklist

- [X] Format with `scripts/format.bash`
- [X] Test locally with `scripts/test_plugin.bash --file plugins/<your_new_plugin_name>`
- [X] Your plugin CI tests are green

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -589,6 +589,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
...
 | tfmigrate                     | [dex4er/asdf-tfmigrate](https://github.com/dex4er/asdf-tfmigrate)                                               | [![CI](https://github.com/dex4er/asdf-tfmigrate/actions/workflows/ci.yml/badge.svg)](https://github.com/dex4er/asdf-tfmigrate/actions/workflows/ci.yml)                                                                           |
+| tfnotify                      | [jnavarrof/asdf-tfnotify](https://github.com/jnavarrof/asdf-tfnotify)                                           |                                                                                                                                                                                                                                   | [![Build Status](https://github.com/jnavarrof/asdf-tfnotify/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/jnvarrof/asdf-tfnotify/actions/workflows/build.yml) |
```
Comment (jthegedus) [inline README.md:592]: You have added an extra column here with `||`, please update, reformat and push

**Review by jthegedus — CHANGES_REQUESTED** [review]
Please fix the extra column

File: README.md (hunk 1 above)
Comment (jnavarrof) [reply README.md]: Thanks. It should be OK now.

**Review by jthegedus — APPROVED** [review]
thanks

**Pull request merged**